pages/ja/lb4/Crafting-LoopBack-4.md

```diff
@@ -0,0 +1,358 @@
+---
+lang: jp
+title: 'LoopBack 4を構築する'
+keywords: LoopBack 4.0, LoopBack 4
```

Comment:
Similarly here, we need `layout: translation`

Reply:
Added  `layout: translation`